gengetopt-toolfile.spec

```diff
@@ -0,0 +1,21 @@
+### RPM external gengetopt-toolfile 1.0
+Requires: gengetopt
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/gengetopt.xml
+<tool name="gengetopt" version="@TOOL_VERSION@">
+  <client>
```

Comment:
Does this package provide any shared library which should be in `LD_LIBRARY_PATH`. If yes, you should add the lines `<lib name="[NAME]"/>` where you have, e.g., `lib[NAME].so`.